core/src/avm2/globals/flash/net/local_connection.rs

```diff
@@ -53,7 +57,24 @@ pub fn send<'gc>(
         )?));
     }
 
-    avm2_stub_method!(activation, "flash.net.LocalConnection", "send");
+    // FIXME: The rest of the logic in this function should actually run
```

Comment:
Since you're using `dispatchStatusEventWithDelay`, this comment is outdated, right?

Reply:
I was referencing the entire rest of the function, which includes checking if there are any active listeners that have called `connect` on the connection. I think I can move that entire logic into a `setTimeout` now.